### PR TITLE
WIP: Draft episode/lesson content

### PR DESCRIPTION
**Pull request by brymz**
Closed with rebase of #14 to #15.

Remaining second-pass edits:

- [x] **bold** important terms
- [x] check for short link - footnote structure
- [x] cite A Abela

### CODE REVIEW

File: _episodes/02-data-management.md [hunk 1]
```diff
@@ -9,119 +9,148 @@ objectives:
 - "To read Gapminder data to R"
 - "To evaluate Gapminder data structure"
 keypoints:
-- "Be sure to `setwd()` to point to your data file before importing it."
+- "Be sure to setup an RStudio project or `setwd()` to point to your data file
```
Comment (gvwilson) [inline _episodes/02-data-management.md:12]: "set up" (but I admit I'm old-fashioned about that)

File: _episodes/02-data-management.md [hunk 2]
```diff
@@ -9,119 +9,148 @@ objectives:
...
+> 6       44.736       45.326       47.360       49.580  2445618  2667518  2961915  3330989  3529983  3834415
+>   pop_1982 pop_1987 pop_1992 pop_1997 pop_2002 pop_2007
+> 1 20033753 23254956 26298373 29072015 31287142 33333216
+> 2  7016384  7874230  8735988  9875024 10866106 12420476
+> 3  3641603  4243788  4981671  6066080  7026113  8078314
+> 4   970347  1151184  1342614  1536536  1630347  1639131
+> 5  6634596  7586551  8878303 10352843 12251209 14326203
+> 6  4580410  5126023  5809236  6121610  7021078  8390505
+> ~~~
+> {: .output}
```
Comment (gvwilson) [inline _episodes/02-data-management.md:79]: Is it worth remarking on the line-wrapping? Or will learners know all about that by this point?

**Comment by gvwilson** [comment]
Some very (very) minor points, but it looks good to me.

**Comment by brymz** [comment]
Thanks for the quick feedback! My plan is to get through each of the episodes and then sweep back for revisions.

**Pull request closed without merging**